vue/src/components/SessionModal.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <modal name="session-modal"
+    transition="pop-out"
+    :height="'auto'"
+    :width="modalWidth"
+    :adaptive="true"
+    :pivotY="0.5"
+    :scrollable="true"
+    :reset="true"
+    @before-open="beforeOpened"
+    @before-close="beforeClosed">
+    <div class="card">
+      <div class="card-content">
+        <h1 class="title is-4">Create Session</h1>
+
+        <!-- Content field -->
+        <div class="field content">
+          <article class="message is-warning">
```

Comment:
Try moving this from an is-warning to an is-info, or removing the message bounds all together into just a paragraph element. I don't know if a warning message is necessary, and my eyes were immediately drawn to it on modal open.